sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/CaseApiService.java

```diff
@@ -47,7 +47,7 @@ public Case buildGeneratedCase(Person person) {
         .hospitalization(Hospitalization.builder().uuid(UUID.randomUUID().toString()).build())
         .symptoms(Symptoms.builder().uuid(UUID.randomUUID().toString()).build())
         .therapy(Therapy.builder().uuid(UUID.randomUUID().toString()).build())
-        .healthFacility(HealthFacility.builder().uuid("SORMAS-CONSTID-ISNONE-FACILITY").build())
+        .healthFacility(HealthFacility.builder().uuid("WYPOCQ-IWVWGQ-XU7YCF-OSQJSAD4").build())
```

Comment:
Does this change impact other tests that are using the cas ecreated through API?

Reply:
Running the API feature returns pass:
![image](https://user-images.githubusercontent.com/46056983/121329519-f5ccb400-c91d-11eb-823a-2c473df354e6.png)
But we can be sure only when having regression.

Reply:
OK, we will see if we need to implement another case bilder.